partiql-lang/src/main/kotlin/org/partiql/lang/CompilerPipeline.kt

```diff
@@ -163,6 +163,8 @@ interface CompilerPipeline {
          */
         fun addFunction(function: ExprFunction): Builder = this.apply { customFunctions[function.signature.name] = function }
```

Comment:
Seemingly, we need to modify the definition of `customFunctions` to not be a map, but to also be a list. We're accidentally overwriting custom functions when they share the same name.